lib/cucumber/wire/protocol/requests.rb

```diff
@@ -92,6 +106,11 @@ def handle_diff(tables)
           def table(data, location)
             Cucumber::MultilineArgument.from_core(Core::Test::DataTable.new(data, location))
           end
+
+          def embeddings(params)
+            return [] if params.nil?
```

Comment:
Minor ruby optimisation here.
`params&.fetch('embeddings') || []`

This is shorthand for `params && params.fetch('embeddings') || []`

Reply:
EDIT: Don't do the top one, just do `params && params.fetch('embeddings') || []` as the abbreviated form requires ruby 2.3 which this gem doesn't enforce directly yet